src/sentry/notifications/notifications/activity/note.py

```diff
@@ -33,3 +37,15 @@ def get_notification_title(
 
     def get_message_description(self, recipient: RpcActor, provider: ExternalProviders) -> Any:
         return self.get_context()["text_description"]
+
+    def description_as_html(self, description: str, params: Mapping[str, Any]) -> SafeString:
```

Comment:
`description` and `params` aren't actually needed here, but that's what the base function accepts. It's funny that we don't need `description` because that's the comment itself, but in the template (`note.html`) it's rendered from `data.text`.